lms/product/plugin/misc.py

```diff
@@ -0,0 +1,26 @@
+class MiscPlugin:
+    """
+    Plugin for product differences that don't have a better abstractions around them.
+
+    Works as a placeholder location to centralize LMS differences here without the need to create
+    bigger abstractions before it's clear what that abstraction should look like.
+
+    The aim also includes to remove any checks around product.family around the code base.
+
+    New methods here should not try to get a very tight API as easier to refactor once multiple
+    LMS have the same issue vs getting the right parameters in the first occurrence.
+
+    Once any of these is implemented by more than one product or a group of
+    methods looks like it could belong to their own plugin it's time to refactor them out.
+    """
```

Comment:
```suggestion
    """
    Plugin for product differences that don't have a better abstractions yet.

    Works as a placeholder location to centralize LMS differences here without
    the need to create bigger abstractions before it's clear what that
    abstraction should look like.

    The aim also includes to remove any checks around `product.family` around
    the code base.

    New methods here should not try to get a very tight API as easier to
    refactor once multiple MS have the same issue vs getting the right
    parameters in the first occurrence.

    Once any of these is implemented by more than one product or a group of
    methods looks like it could belong to their own plugin it's time to
    refactor them out.
    """
```

Nit-picky 79 char pep8 stuff. I have a grey line in my IDE that shows this limit which is quite helpful.

Apart from anything else it ensures it will show up nicely in Github without line wrapping as you can see above.